backend/deepchecks_monitoring/api/v1/data_ingestion_alert.py

```diff
@@ -0,0 +1,120 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+"""V1 API of the alerts."""
+import typing as t
+
+import pendulum as pdl
+from fastapi import Depends, Response, status
+from pydantic import BaseModel
+from sqlalchemy import false, func, select, update
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from deepchecks_monitoring.config import Tags
+from deepchecks_monitoring.dependencies import AsyncSessionDep
+from deepchecks_monitoring.public_models.user import User
+from deepchecks_monitoring.schema_models.data_ingestion_alert import DataIngestionAlert
+from deepchecks_monitoring.schema_models.data_ingestion_alert_rule import DataIngestionAlertRule
+from deepchecks_monitoring.schema_models.model import Model
+from deepchecks_monitoring.schema_models.model_memeber import ModelMember
+from deepchecks_monitoring.utils import auth
+from deepchecks_monitoring.utils.alerts import AlertSeverity
+
+from .router import router
+
+
+class DataIngestionAlertCreationSchema(BaseModel):
```

Comment:
Why do you need a creation schema? Are clients allowed to create alerts manually?